neuralmonkey/nn/projection.py

```diff
@@ -19,9 +19,12 @@ def linear(inputs, size, scope="LinearProjection"):
         A tensor of shape batch x size
     """
     with tf.variable_scope(scope):
-        return tf.nn.seq2seq.linear(inputs, size, True)
+        # TODO this will be soon deprecated, should be rewrited with
+        # tf.contrib.layers.fully_connected and a support for list of tensors
+        return tf.nn.seq2seq.linear(inputs, size, True, scope=scope)
 
 
+# SUGGESTION: what about using ReLu as a default? It have better results
```

Comment:
Proč by to měl kvůli lepším výsledkům být default? Tady bych spíš vůbec žádný default nevyplňoval.